### PR TITLE
Issue #4387: added sevntu no error test

### PR DESCRIPTION
**Pull request by rnveach**
Issue #4387 

When I tested this locally I didn't get any exception and I'm not sure why. Wercker is the same.

### CODE REVIEW

**Comment by rnveach** [comment]
I see reason now.
sevntu error is because we don't specify full paths in checkstyle's `checkstyle_sevntu_checks.xml`.
The reason we didn't catch this in PR is because [the sevntu checkstyle run uses checkstyle release instead of snapshot](https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L467-L471). 

We would have to install SNAPSHOT and then run sevntu's check to be able to test this.

**Comment by rnveach** [comment]
new wercker item now fails.

File: pom.xml [hunk 1]
```diff
@@ -467,7 +468,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>7.8</version>
+            <version>${maven.checkstyle.plugin.version.sevntu}</version>
```
Comment (romani) [inline pom.xml]: Might be better to name - maven.sevntu-checkstyle-check.checkstyle.version

Comment (rnveach) [inline pom.xml]: Done.

**Comment by rnveach** [comment]
See discussion at https://github.com/checkstyle/checkstyle/pull/4389#pullrequestreview-41446468 .
This PR might not be needed.

**Comment by rnveach** [comment]
So this PR is still on.
Are we ok to merge?

**Comment by romani** [comment]
ok to merge.

**Pull request merged**